solutions/USACO feb16-milk-pails.mdx

```diff
@@ -53,4 +53,56 @@ int main () {
 ```
 
 </CPPSection>
+<JavaSection>
```

Comment:
I think it would be nice to have an extra line here. (blank line between `</CPPSection>` and `<JavaSection>`)

Reply:
Between code blocks and the code, though, is better without extra newline (so what you have below is good)

Reply:
By "below," i mean this part:

```
```java
import java.util.*;
```

Reply:
Sounds good I'll keep that in mind